test/engine.ts

```diff
@@ -4,8 +4,12 @@ import resolve from '../src/engine'
 describe('Engine', () => {
   describe('.resolve', () => {
     it('returns ResolvedEngine class with resolved class', async () => {
-      expect((await resolve(Marp)).klass).toBe(Marp)
-      expect((await resolve('@marp-team/marp-core')).klass.name).toBe('Marp')
```

Comment:
The name of `Marp` class provided by Marp Core v1.0.1 has applied minification by bumped terser.

Marp CLI will use name defined in `package.json` but won't use engine name. We updated test case, to confirm matching a resolved class with imported Marp Core.